test/builders/models/otheremployment/ShareAwardedOrReceivedBuilder.scala

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package builders.models.otheremployment
+
+import models.otheremployment.ShareAwardedShareSchemePlanType.SIP
+import models.otheremployment.SharesAwardedOrReceived
+
+import java.time.LocalDate
+
+object ShareAwardedOrReceivedBuilder {
+
+  val aSharesAwardedOrReceived: SharesAwardedOrReceived = SharesAwardedOrReceived(
+    employerName = "ABC Ltd",
+    employerRef = Some("321/AB156"),
+    schemePlanType = SIP,
+    dateSharesCeasedToBeSubjectToPlan = LocalDate.parse("2019-10-20"),
```

Comment:
Instead of parsing a string which can cause exceptions, could you use the more typesafe LocalDate.of instead?

Reply:
We had this warnings of using magic numbers if we used that method. Do you remember if we ever got an alternative to that?

Reply:
You can import the Months you need, but I don't know what to do about the day and year numbers. We're going to have hard-coded values whether integers or string literals, so the hard-coded-ness is there in both cases. I would say it is more type-safe to use integers so it is better to use them and incur (or ignore) any warnings